symphonia-codec-opus/Cargo.toml

```diff
@@ -14,4 +14,19 @@ rust-version = "1.53"
 
 [dependencies]
 log = "0.4"
-symphonia-core = { version = "0.5.4", path = "../symphonia-core" }
\ No newline at end of file
+symphonia-core = { version = "0.5.4", path = "../symphonia-core" }
+once_cell = "1.19.0"
+lazy_static = "1.5.0"
```

Comment:
Use `lazy_static` only in `no_std` enviroment.

Reply:
Removed 3c92322 It was unused dep